package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "tslint --fix -c tslint.json -t stylish -p ./tsconfig.json",
     "update": "npx npm-check -u",
     "prepublishOnly": "npm run build",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "tap -j 4 src/flash.test.ts"
```

Comment:
I recommend you to use `-J` instead of `-j4`, it will use all of the core available in the machine :)